### PR TITLE
Digitrust id integration

### PR DESCRIPTION
**Pull request by goosemanjack**
## Type of change
Integration of support for DigiTrust Id within the new UserId module for Prebid.
- [ X] Feature

## Description of change
Integration of support for DigiTrust Id within the new UserId module for Prebid.
This module allows the DigiTrust Id to be accessed by compliant bidder-adapters
via the UserId system.

- contact email of the adapter’s maintainer
sam@iabtechlab.com
ccole@emination.com

- [X ] official adapter submission

## Other information
Full instructions linked to from the associated Markdown page.
More information on DigiTrust, source code, and implementation guide
may be found at:

* http://www.digitru.st/
* https://github.com/digi-trust/dt-cdn
* https://github.com/digi-trust/dt-cdn/wiki

DigiTrust is now a part of IAB and maintained by IAB Tech Lab.
https://iabtechlab.com/

### CODE REVIEW

**Comment by goosemanjack** [comment]
Any review status on this?

**Comment by bretg** [comment]
Were you going to draft the proposed changes to http://prebid.org/dev-docs/modules/userId.html and http://prebid.org/dev-docs/modules/digitrust.html ?

**Comment by goosemanjack** [comment]
I could certainly make changes to those docs. Where would I find them and gain access? You can send in email if needed.

**Comment by samtingleff** [comment]
@goosemanjack https://github.com/prebid/prebid.github.io/blob/master/dev-docs/modules/userId.md and https://github.com/prebid/prebid.github.io/blob/master/dev-docs/modules/digitrust.md

**Comment by goosemanjack** [comment]
I have made the documentation changes to the prescribed repo. There is a pull request for it:
[Pull Request 1298 on prebid.github.io](https://github.com/prebid/prebid.github.io/pull/1298)

File: modules/digiTrustIdLoader.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    utils.logError('DigiTrustLoader failed to find Prebid global');
+    return;
+  }
+
+  config.setConfig({ digitrustId: true });
+
+  if (pbjs.idsys == null) {
+    pbjs.idsys = {};
+  }
+  pbjs.idsys[digitrustIdModule.name] = digitrustIdModule;
```
Comment (idettman) [inline modules/digiTrustIdLoader.js]: for the sub-module registration, can you use `attachIdSystem` from the UserId Module, for example:
```
// add import
import {attachIdSystem} from './userId.js';

// add submodule to the userId module before the setConfig for userSync.userIds is set (ideally when the .js file inits)
attachIdSystem(digitrustIdModule);
```

Comment (goosemanjack) [reply modules/digiTrustIdLoader.js]: Yes, I'm in the process of reworking. I'm going to submit a fresh PR once ready.

**Pull request closed without merging**